contrib/apiage.py

```diff
@@ -311,6 +383,14 @@ def main():
             sys.exit(1)
         write_json(cli.current, api_tracked)
         write_markdown(cli.document, api_tracked)
+    elif cli.mode == "fix-versions":
+        values = {}
+        _setif(values, "added_in_version", cli.added_in_version)
+        _setif(values, "expected_stable_version", cli.stable_in_version)
+        _setif(values, "deprecated_in_version", cli.deprecated_in_version)
+        _setif(values, "expected_remove_version", cli.remove_in_version)
```

Comment:
I couldn't see default for `cli.remove_in_version` in __setif()_. Was that intentionally left out?

Reply:
Blame my natural inconsistency or my desire to use closer-to-normal english grammar. The CLI option is called `--remove-in-version` (which argparse turns into `remove_in_version`) since you are *telling* the script since your telling the script what version the API(s) are to be removed. The json key is called "expected_remove_version" as it's a guess about the future. Does that clear it up?

Reply:
That's all fine. I understood.

I think I put it wrong previously that assigning default for `cli.remove_in_version`(=>`expected_remove_versions`) was not present in __setif()_. Actually I was pointing at _tag_to_versions()_ where we don't check for `--remove-in-version` or set default based on `--current-tag`.

Reply:
Oh, are you asking why there's no setting of remove_in_version from tag_to_version based on the current git tag?  If so, that's because there's no policy about when to actually remove a deprectated API. Since there's no policy if you want to set it you have to set it manually. Does that make sense?

Reply:
Yes.